compass/ocean/tests/parabolic_bowl/parabolic_bowl.cfg

```diff
@@ -0,0 +1,54 @@
+[job]
+
+wall_time = 2:00:00
+
+
+# config options for drying slope test cases
+[parabolic_bowl]
+
+# dimensions of domain in x and y directions (km)
+Lx = 1440
+Ly = 1560
+
+# Coriolis parameter
+coriolis_parameter = 1.031e-4
+
+# Maximum initial ssh magnitude
+eta_max = 2.0
+
+# Maximum water depth
+depth_max = 50.0
+
+# Angular fequency of oscillation
+omega = 1.4544e-4
+
+# Gravitational acceleration
+gravity = 9.81
+
+# Number of tasks for resolutions
+20km_ntasks = 72
+20km_min_tasks = 4
+
+10km_ntasks = 72
+10km_min_tasks = 4
+
+5km_ntasks = 128
+5km_min_tasks = 4
```

Comment:
We could probably leave this out since in polaris we will be using these cfg options to determine the number of tasks:
```
# the number of cells per core to aim for
goal_cells_per_core = 200

# the approximate maximum number of cells per core (the test will fail if too
# few cores are available)
max_cells_per_core = 2000